components/brave_wallet_ui/components/desktop/popup-modals/add-account-modal/add-imported-account-modal.tsx

```diff
@@ -74,8 +74,12 @@ export const ImportAccountModal = () => {
   }, [accountTypeName, isFilecoinEnabled, isSolanaEnabled])
 
   // state
+  const selectedFilecoinNetwork = useSelector(({ wallet }: { wallet: WalletState }) => {
+    return wallet.defaultNetworks.find((network) => { return network.coin === BraveWallet.CoinType.FIL })
+})
   const [accountName, setAccountName] = React.useState<string>('')
-  const [filecoinNetwork, setFilecoinNetwork] = React.useState<FilecoinNetwork>('f')
+  const networkSymbol = selectedFilecoinNetwork?.chainId === BraveWallet.FILECOIN_MAINNET ? BraveWallet.FILECOIN_MAINNET : BraveWallet.FILECOIN_TESTNET
```

Comment:
Can we compare lowercase values i.e `selectedFilecoinNetwork?.chainId.toLowerCase() === BraveWallet.FILECOIN_MAINNET.toLowerCase()`

Reply:
done